R/replay.r

```diff
@@ -77,7 +113,14 @@ line_prompt <- function(x, prompt = getOption("prompt"), continue = getOption("c
   lines <- strsplit(x, "\n")[[1]]
   n <- length(lines)
 
-  lines[1] <- str_c(prompt, lines[1])
```

Comment:
In the spirit of "do one thing and do it well", I'd like these changes to go to `replay.source()` instead of `line_prompt()`, otherwise `line_prompt()` is doing two things.